src/domains/transaction/components/FilterTransactions/FilterTransactions.test.tsx

```diff
@@ -8,6 +8,18 @@ import { env, getDefaultProfileId, render, screen } from "@/utils/testing-librar
 let profile: Contracts.IProfile;
 
 describe("FilterTransactions", () => {
+	const allTypes = [
+		"delegateRegistration",
+		"delegateResignation",
+		"ipfs",
+		"multiPayment",
+		"multiSignature",
+		"secondSignature",
+		"transfer",
+		"vote",
+		"magistrate",
```

Comment:
why do the tests include all of these types when only a subset is in the actual filter now?